tutorials/sklearn-integration/H2OAutoML_as_sklearn_estimator.ipynb

```diff
@@ -0,0 +1,529 @@
+       "<tr><td>H2O cluster name:</td>\n",
+       "<td>H2O_from_python_seb_ca9mh4</td></tr>\n",
+       "<tr><td>H2O cluster total nodes:</td>\n",
+       "<td>1</td></tr>\n",
+       "<tr><td>H2O cluster free memory:</td>\n",
+       "<td>3.048 Gb</td></tr>\n",
+       "<tr><td>H2O cluster total cores:</td>\n",
+       "<td>8</td></tr>\n",
+       "<tr><td>H2O cluster allowed cores:</td>\n",
+       "<td>8</td></tr>\n",
+       "<tr><td>H2O cluster status:</td>\n",
+       "<td>locked, healthy</td></tr>\n",
+       "<tr><td>H2O connection url:</td>\n",
+       "<td>http://localhost:54321</td></tr>\n",
+       "<tr><td>H2O connection proxy:</td>\n",
+       "<td>{'http': None, 'https': None}</td></tr>\n",
+       "<tr><td>H2O internal security:</td>\n",
+       "<td>False</td></tr>\n",
+       "<tr><td>H2O API Extensions:</td>\n",
+       "<td>Amazon S3, XGBoost, Algos, AutoML, Core V3, TargetEncoder, Core V4</td></tr>\n",
+       "<tr><td>Python version:</td>\n",
+       "<td>3.7.5 final</td></tr></table></div>"
+      ],
+      "text/plain": [
+       "--------------------------  ------------------------------------------------------------------\n",
+       "H2O cluster uptime:         1 hour 17 mins\n",
+       "H2O cluster timezone:       Europe/Prague\n",
+       "H2O data parsing timezone:  UTC\n",
+       "H2O cluster version:        3.28.0.2\n",
+       "H2O cluster version age:    22 hours and 54 minutes\n",
+       "H2O cluster name:           H2O_from_python_seb_ca9mh4\n",
+       "H2O cluster total nodes:    1\n",
+       "H2O cluster free memory:    3.048 Gb\n",
+       "H2O cluster total cores:    8\n",
+       "H2O cluster allowed cores:  8\n",
+       "H2O cluster status:         locked, healthy\n",
+       "H2O connection url:         http://localhost:54321\n",
+       "H2O connection proxy:       {'http': None, 'https': None}\n",
+       "H2O internal security:      False\n",
+       "H2O API Extensions:         Amazon S3, XGBoost, Algos, AutoML, Core V3, TargetEncoder, Core V4\n",
+       "Python version:             3.7.5 final\n",
+       "--------------------------  ------------------------------------------------------------------"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Parse progress: |█████████████████████████████████████████████████████████| 100%\n",
+      "Parse progress: |█████████████████████████████████████████████████████████| 100%\n",
+      "AutoML progress: |██████████\n",
+      "17:56:49.537: Skipping training of model GBM_5_AutoML_20200121_175640 due to exception: water.exceptions.H2OModelBuilderIllegalArgumentException: Illegal argument(s) for GBM model: GBM_5_AutoML_20200121_175640.  Details: ERRR on field: _min_rows: The dataset size is too small to split for min_rows=100.0: must have at least 200.0 (weighted) rows, but have only 112.0.\n",
```

Comment:
@sebhrusen Can we use a bigger dataset that does not produce errors?

Reply:
just replaced `iris` with `breast-cancer` dataset example from `sklearn`: no error and better perf of SEs